### PR TITLE
rename ompslp_g ompslpnc for clarity and IODA converter compliance

### PR DESCRIPTION
**Pull request by karpob**
This a 3 part update to replace ompslp_g with ompslpnc in all places. 1/3

### CODE REVIEW

**Review by kwargan — DISMISSED** [review]

File: GMAO_etc/obsys-nccs-geosit.rc [hunk 1]
```diff
@@ -423,11 +423,67 @@ BEGIN ncep_cris_bufr => gdas1.%y4%m2%d2.t%h2z.cris.tm00.bufr_d.ublk
   20120106_06z-20120112_18z 060000 /discover/nobackup/projects/gmao/input/dao_ops/ops/flk/ncep_g5obs/bufr/CRIS/Y%y4/M%m2/gdas1.%y4%m2%d2.t%h2z.cris.tm00.bufr_d
   20120406_12z-20200422_18z 060000 /discover/nobackup/projects/gmao/input/dao_ops/ops/flk/ncep_g5obs/bufr/CRIS/Y%y4/M%m2/gdas1.%y4%m2%d2.t%h2z.cris.tm00.bufr_d
 END
+<<<<<<< Updated upstream
```
Comment (mathomp4) [inline GMAO_etc/obsys-nccs-geosit.rc:426]: @karpob Something bad happened here. You have conflict markers in at least one of these files.

Comment (mathomp4) [reply GMAO_etc/obsys-nccs-geosit.rc:426]: Indeed, there are conflict markers in *all* the files. It looks like a merge went bad.

Comment (karpob) [reply GMAO_etc/obsys-nccs-geosit.rc:426]: Ah, yeah. Sorry about that. I was trying to get this out before leaving for the weekend.

Comment (mathomp4) [reply GMAO_etc/obsys-nccs-geosit.rc]: Ah. Looks good now. 

Well, looks like there are no conflict markers. What these files actually do are mysteries to me so... 🤷🏼  😄

**Comment by karpob** [comment]
I think there are issues with how I'm doing this. I think I'm trying to update based off the tag that's in GEOSadas, and that isn't going to work right

**Pull request closed without merging**